buildenv/jenkins/openjdk_s390x_zos.xml

```diff
@@ -0,0 +1,22 @@
+#!groovy
+/* Template for test jobs on z/OS. Configure test job as parameterized.
+	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
+	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_s390x)
+	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
+LABEL='ci.project.openj9&&hw.arch.s390&&hw.arch.z14&&sw.os.zos'

```

Comment:
- label is updated. `hw.arch.s390` will not have match. Please change it to `hw.arch.s390x`
- Please remove `ci.project.openj9`
- do we need to have `hw.arch.z14`?

Reply:
does the matched machines look right to you?
https://hyc-runtimes-jenkins.swg-devops.com/label/hw.arch.s390x&&sw.os.zos/

Reply:
Since I am not working on labels much, and they are being changed, I am not an expert on how to validate which combination of labels will find a match. For example I did not know you could use this URL to validate a combination. Thanks for pointing it out. Since you know how to confirm a combination - do you have an answer to your question - "do we need to have hw.arch.z14?" - ?

Reply:
Thanks for letting me know there is this easy way to validate combination of labels. The matched machines are right.